Kiosk/Views/SettingsPage.xaml

```diff
@@ -121,6 +121,27 @@
 
                     <PasswordBox Header="Translator Text API Key" Password="{Binding TranslatorTextApiKey, Mode=TwoWay}" Margin="0,10,0,0" />
 
+                    <Grid Margin="0,10,0,0">
+                        <Grid.ColumnDefinitions>
+                            <ColumnDefinition/>
+                            <ColumnDefinition Width="auto"/>
+                            <ColumnDefinition Width="auto"/>
+                        </Grid.ColumnDefinitions>
+                        <PasswordBox Header="Ink Recognizer API Key" Password="{Binding InkRecognizerApiKey, Mode=TwoWay}" />
+                        <ComboBox Grid.Column="1" Margin="10,0,6,0" Header="Endpoint" Width="355"
+                                  x:Name="inkRecognizerEndpointCombobox"
+                                  ItemsSource="{Binding InkRecognizerEndpoints}"
+                                  SelectedItem="{Binding BindingInkRecognizerApiKeyEndpoint, Mode=TwoWay}">
+                        </ComboBox>
+                        <StackPanel Grid.Column="2" Orientation="Horizontal" 
+                                    Visibility="{Binding ElementName=inkRecognizerEndpointCombobox, Path=SelectedValue, Converter={StaticResource StringMatchToVisibilityConverter}, ConverterParameter={StaticResource CustomEndpointName}}">
+                            <TextBox Header="Custom Text Analytics Endpoint" MaxWidth="450" x:Name="inkRecognizerApiEndpointTextBox"
+                                     PlaceholderText="Custom endpoint (e.g. https://inkreco.cognitiveservices.azure.com)"
```

Comment:
put a legit url for the placeholder?  or something that is clearly not real like <enter endpoint here> ..